api.planx.uk/lib/hasura/metadata/index.test.ts

```diff
@@ -1,32 +1,40 @@
 import { createScheduledEvent, RequiredScheduledEventArgs } from "./index.js";
-import Axios, { AxiosError } from "axios";
+import axios from "axios";
+import type { Mocked } from "vitest";
 
-jest.mock("axios", () => ({
-  ...jest.requireActual("axios"),
-  post: jest.fn(),
-}));
-const mockAxios = Axios as jest.Mocked<typeof Axios>;
+describe("Creation of scheduled event", () => {
+  vi.mock("axios", async (importOriginal) => {
```

Comment:
vitest has this `importOriginal` argument which is passed to the factory constructor in a `mock` call - we can use this to import the actual module (replacing `jest.requireActual`), as per [the docs](https://vitest.dev/guide/migration.html#importing-the-original-of-a-mocked-package).

Could also have used `vi.importActual` here, but using the arg seemed neater.